Xwt.XamMac/Xwt.Mac/WindowBackend.cs

```diff
@@ -211,33 +210,45 @@ void IBackend.EnableEvent (object eventId)
 			if (eventId is WindowFrameEvent) {
 				var @event = (WindowFrameEvent)eventId;
 				switch (@event) {
-				case WindowFrameEvent.BoundsChanged:
```

Comment:
Shown/Hidden uses observers and this uses selectors. Should we unify on one pattern? There's DidResizeNotification and DidMoveNotification.

Reply:
Using selectors with the delegate is cleaner imo, so I'd prefer keeping it. Unfortunately we can't change thjat for the observer cos we observe ContentView and not the window.